### PR TITLE
`avn cloud` documentation 

### PR DESCRIPTION
**Pull request by ftisiot**
Added the file `docs/tools/cli/cloud.rst` documenting the `avn cloud`.
Pretty simple and based on what @lornajane did for `avn project`

### CODE REVIEW

File: docs/tools/cli.rst [hunk 1]
```diff
@@ -63,6 +63,8 @@ Manage the payment cards on your account.
```
Comment (hab-anka) [inline docs/tools/cli.rst:63]: Could we also make the `cloud `heading linked?

**Review by hab-anka — APPROVED** [review]
A few small suggestions (link, short desc and punctuation) - some in a commit.

File: docs/tools/cli.rst (hunk 1 above)
Comment (ftisiot) [reply docs/tools/cli.rst:63]: If we do so we have to change all the titles from being code like ``cloud`` to cloud. @lornajane what do you think?

Comment (ftisiot) [reply docs/tools/cli.rst:63]: The drawback if we don't link from the title is that we'll need to add aSee detailed command information .... for each command.. that could become quite repetitive

Comment (lornajane) [reply docs/tools/cli.rst:63]: Let's leave it as it is. Linking titles isn't a common UX pattern (or the links would be internal ones to let you link directly to the section, not to somewhere else) and I think people should read the high level information and then decide if they need more info.

**Review by lornajane — APPROVED** [review]
Small but mighty!

**Pull request merged**